site/static/examples/sample-validating-policy-binding.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: admissionregistration.k8s.io/v1
+kind: ValidatingAdmissionPolicyBinding
+metadata:
+  name: sample-validating-admission-policy-binding
+spec:
+  policyName: sample-validating-admission-policy
+  validationActions: [Deny]
+  matchResources:
+    namespaceSelector:
+      matchExpressions:
+        - key: kubernetes.io/metadata.name
+          operator: NotIn
+          values: ["admin-namespace"]
```

Comment:
@dgrove-oss Is the namespace in openshift kueue-system?
If so, I think we can include it as part of the values here.

Reply:
When Kueue gets deployed as part of OpenShift AI, it is put in the `redhat-ods-application` namespace.  I think probably best to just leave as the generic admin-namespace.

@IrvingMg -- thanks for writing this up; this is great!

Reply:
btw, I tested using validating admission policies today in our [setup](https://github.com/project-codeflare/mlbatch/blob/84610a12ba61cc1d05711c2da1f97b685f98ad61/setup.k8s-v1.30/admission-policy.yaml) (now open sourced).  It worked very nicely.
We use a label on "user" namespaces to identify where we want the policy to be enforced.

Reply:
Great to hear!

Btw, have you considered adding yourselves to https://kueue.sigs.k8s.io/docs/adopters/?